internal/services/authorization/pim_eligible_role_assignment_resource_test.go

```diff
@@ -121,6 +121,22 @@ func TestAccPimEligibleRoleAssignment_expirationByDate(t *testing.T) {
 	})
 }
 
+func TestAccPimEligibleRoleAssignment_condition(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_pim_eligible_role_assignment", "test")
+	r := PimEligibleRoleAssignmentResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.condition(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("condition").Exists(),
+				check.That(data.ResourceName).Key("condition_version").HasValue("2.0"),
+			),
+		},
```

Comment:
Can we update this test to use the `data.ImportStep()` instead of checking the specific values?